### DISCUSSION TITLE
ScriptReference/CompositeCollider2D

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#633] [Notes]
# ScriptReference/CompositeCollider2D

https://docs.unity3d.com/ScriptReference/CompositeCollider2D.html

**Comment by qwe321qwe321qwe321** [comment]
[Put a disabled collider into CompositeCollider2D by script doesn't work.](https://forum.unity.com/threads/put-a-disabled-collider-into-compositecollider2d-by-script-doesnt-work.1208362/)

**Comment by qwe321qwe321qwe321** [comment]
[Composite Collider doesn't work with polygon collider which has scale.x = -1](https://forum.unity.com/threads/composite-collider-doesnt-work-with-polygon-collider-which-has-scale-x-1.940158)
[COMPOSITE COLLIDER DOESN'T WORK WITH POLYGONCOLLIDER2D WITH INVERTED SCALE.](https://issuetracker.unity3d.com/issues/composite-collider-doesnt-work-with-polygoncollider2d-with-inverted-scale) (Fixed in 2021.1)
